internal/experiment/generation/generation.go

```diff
@@ -97,10 +97,16 @@ func loadApplicationData(app *redskyappsv1alpha1.Application, src string) ([]byt
 
 	opts := []getter.ClientOption{
 		func(c *getter.Client) error {
```

Comment:
I can put this as a separate PR if needed since its unrelated to this. But without this change, we don't work.

Reply:
This is fixed in the latest #288 changes, along with a comment clarifying why we _do not_ want use the process working directory here.